packages/create-vite/template-lit-ts/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "vite-lit-ts-starter",
   "private": true,
   "version": "0.0.0",
+  "type": "module",
   "main": "dist/my-element.es.mjs",
```

Comment:
Should we also use `dist/my-element.js` now that we type module? I think it is better if the templates already provide a clean world for ES, and add extra noise for CJS.

Reply:
So basically zero cjs

Reply:
Looks like we need to change this once we switch the templates to Vite 3, as this has been changed by https://github.com/vitejs/vite/pull/8107